src/quacc/settings.py

```diff
@@ -186,11 +185,10 @@ class QuaccSettings(BaseSettings):
     ESPRESSO_PSEUDO: Optional[Path] = Field(
         None, description=("Path to a pseudopotential library for espresso.")
     )
-    ESPRESSO_PRESET_DIR: Path = Field(
-        Path(__file__).parent / "calculators" / "espresso" / "presets",
+    ESPRESSO_PRESET_DIR: list[Path] = Field(
```

Comment:
For here and `VASP_PRESET_DIR`, is it possible for us to accept `Union[Path, list[Path]]` to maintain backwards compatibility?